src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (newRobot.hasOwnProperty(value)) {
+      return null;
+    } else {
+      newRobot[value] = key;
+    }
```

Comment:
You should avoid unnecessary nesting, you can return null in inverted if block and assign values after that block